service/scripts/loadBundles.ts

```diff
@@ -0,0 +1,150 @@
+import * as fs from 'fs';
+import path from 'path';
```

Comment:
This file should probably be removed, but am interested in your thoughts on how:

> Note: I've left the loadBundles bundle loader script piece in for now, but I think we should potentially remove that since other functionality has been added that takes its place. We may want to keep it for the moment ahead of the connectathon in case it's useful (maybe we pull it off into another branch? or maybe just make sure the commit isn't deleted?).

Do you think it would be better to add a commit to this PR removing it (but make sure that the branch doesn't get deleted when we merge so that the commit history of it remains in case we need it)? Or do you think we should move the loadBundles stuff to another branch?

Reply:
Added to branch here instead: https://github.com/projecttacoma/measure-repository/tree/connectathon-loadBundles